pwa/src/templates/templateParts/footer/FooterTemplate.tsx

```diff
@@ -1,58 +1,124 @@
-import { PageFooter, Link } from "@utrecht/component-library-react/dist/css-module";
+import { PageFooter, Link, Heading3 } from "@utrecht/component-library-react/dist/css-module";
 import { navigate } from "gatsby-link";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faCode } from "@fortawesome/free-solid-svg-icons";
 import { useTranslation } from "react-i18next";
 
+type TDynamicContentItem = {
+  title: string;
+  items: { label: string; value: string; link?: string }[];
+};
+
 export const FooterTemplate: React.FC = () => {
-  const { t } = useTranslation();
+  const [footerContent, setFooterContent] = React.useState<TDynamicContentItem[]>([]);
+
+  React.useEffect(() => {
+    if (!process.env.GATSBY_FOOTER_CONTENT) return;
+
+    try {
+      setFooterContent(JSON.parse(process.env.GATSBY_FOOTER_CONTENT));
+    } catch {
+      console.warn("Could not parse footer content.");
+    }
+  }, [process.env.GATSBY_FOOTER_CONTENT]);
 
   return (
     <PageFooter className={styles.footer}>
       <div className={styles.container}>
-        <div className={styles.imageContainer}>
-          {process.env.GATSBY_FOOTER_LOGO_URL !== "false" && (
-            <img
-              className={styles.image}
-              onClick={() =>
-                process.env.GATSBY_FOOTER_LOGO_HREF ? open(process.env.GATSBY_FOOTER_LOGO_HREF) : navigate("/")
-              }
-              src={process.env.GATSBY_FOOTER_LOGO_URL}
-              alt={"Footer-logo"}
-            />
-          )}
+        <div className={styles.contentGrid}>
+          {footerContent?.map((content, idx) => (
+            <DynamicSection key={idx} {...{ content }} />
+          ))}
         </div>
-        <div>
-          <Link
-            className={styles.link}
-            href="https://github.com/ConductionNL/woo-website-template"
-            target="_blank"
-            description={t("Link to github repository")}
-          >
-            <FontAwesomeIcon icon={faCode} />
-          </Link>{" "}
-          with{" "}
-          <Link
-            className={styles.link}
-            href="https://github.com/ConductionNL/woo-website-template/graphs/contributors"
-            target="_blank"
-            name={t("Link to github contributors page")}
-          >
-            ❤️
-          </Link>{" "}
-          by{" "}
-          <Link
-            className={styles.link}
-            href="https://conduction.nl"
-            target="_blank"
-            description={t("Link to conduction website")}
-          >
-            Conduction.
-          </Link>
+
+        <div className={styles.logoAndConduction}>
+          <Logo />
+          <WithLoveByConduction />
         </div>
       </div>
     </PageFooter>
   );
 };
+
+const DynamicSection: React.FC<{ content: TDynamicContentItem }> = ({ content }) => {
+  return (
+    <section>
+      <Heading3 className={styles.dynamicSectionTitle}>{content.title}</Heading3>
```

Comment:
In some themes the Header will not have the same color as the footer color and will look out of place.
![image](https://github.com/ConductionNL/woo-website-template/assets/43807324/32e84ebd-5b6f-4a86-b751-a61f3e1cea8a)

To make sure that the footer is consistent I suggest adding the color as CSS property like so.
```
.dynamicSectionTitle {
  margin-block-end: 18px;
  color: var(--utrecht-page-footer-color);
}
```